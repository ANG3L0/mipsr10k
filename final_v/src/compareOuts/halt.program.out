Chronologic VCS simulator copyright 1991-2011
Contains Synopsys proprietary information.
Compiler version E-2011.03_Full64; Runtime version E-2011.03_Full64;  Dec 10 00:51 2012
@@
@@
@@                     0  Asserting System reset......
@@
@@                     0 : System STILL at reset, can't show anything
@@
@@
@@                    10 : System STILL at reset, can't show anything
@@
@@
@@                    20 : System STILL at reset, can't show anything
@@
@@@ Unified Memory contents hex on left, decimal on right: 
@@@
@@@ mem[    0] = 0000000000000555 : 1365
@@@
@@@
@@                    72 : System halted
@@
@@@ System halted on HALT instruction
@@@
@@
@@  6 cycles / 1 instrs = 6.000000 CPI
@@
@@  150.00 ns total time to execute
@@

$finish called from file "testbench/pipeline_test.v", line 817.
$finish at simulation time                   72
           V C S   S i m u l a t i o n   R e p o r t 
Time: 72
CPU Time:      0.280 seconds;       Data structure size:   0.3Mb
Mon Dec 10 00:51:19 2012
