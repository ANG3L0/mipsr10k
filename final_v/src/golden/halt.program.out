Chronologic VCS simulator copyright 1991-2012
Contains Synopsys proprietary information.
Compiler version G-2012.09_Full64; Runtime version G-2012.09_Full64;  Dec 10 00:19 2012
@@
@@
@@                     0  Asserting System reset......
@@
@@                     0 : System STILL at reset, can't show anything
@@
@@
@@                   100 : System STILL at reset, can't show anything
@@
@@
@@                   200 : System STILL at reset, can't show anything
@@
@@
@@                   300 : System STILL at reset, can't show anything
@@
@@                   360  Deasserting System reset......
@@
@@
@@@ Unified Memory contents hex on left, decimal on right: 
@@@
@@@ mem[    0] = 0000000000000555 : 1365
@@@
@@@
@@                  1420 : System halted
@@
@@@ System halted on HALT instruction
@@@
@@
@@  11 cycles / 0 instrs = inf CPI
@@
@@  300.00 ns total time to execute
@@

$finish called from file "testbench/testbench.v", line 290.
$finish at simulation time                 2420
           V C S   S i m u l a t i o n   R e p o r t 
Time: 242000 ps
CPU Time:      0.310 seconds;       Data structure size:   0.1Mb
Mon Dec 10 00:19:30 2012
